src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,29 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <>
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            className={tab.id === selectedTab.id ? 'is-active' : null}
```

Comment:
yeah, use `classnames` library here please

and add `'is-active'` by using just this condition `tab.id === selectedTab.id `